.gitignore

```diff
@@ -0,0 +1,8 @@
+.DS_Store
```

Comment:
We should not ignore files that are specific to the OS or IDE. Each dev should be responsible for ignoring these files.